src/Microsoft.Extensions.Logging.EventLog/EventLoggerFactoryExtensions.cs

```diff
@@ -44,7 +44,7 @@ public static ILoggerFactory AddEventLog(this ILoggerFactory factory, LogLevel m
         }
 
         /// <summary>
-        /// Adds an event logger that is enabled for <see cref="LogLevel"/>s of minLevel or higher.
+        /// Adds an event logger. Use <paramref name="settings"/> to enabled for specific <see cref="LogLevel"/>s.
```

Comment:
Can you rephrase to `Use <paramref name="settings"/> to enable logging for specific <see cref="LogLevel"/>s.`